blank-canvas-3/theme.json

```diff
@@ -0,0 +1,360 @@
+{
+	"customTemplates": [
```

Comment:
I think the `customTemplates` config can be removed since there is no longer a blank template supplied by the theme.